### DISCUSSION TITLE
Codespaces deny permission to push to source branches of PRs

### DISCUSSION THREAD

**Discussion by adamlui** [community/community#112419] [Codespaces]
### Select Topic Area

Bug

### Body

When viewing a PR > Files changed, then clicking Review in codespace > Create new codespace, changes made cannot be pushed to fork, even though original contributor did not uncheck "Allow edits by maintainer"

![image](https://github.com/community/community/assets/10906554/b9ff052c-a432-4696-bcc0-fb8059b433c4)

Only when editing files using regular github.com editor is permission not denied

**Comment by njzjz** [comment]
I also encountered this error. The workaround is to use the SSH keys to push via SSH instead of using HTTPS and GITHUB_TOKEN.
